src/utils/gitUtils.ts

```diff
@@ -63,10 +64,24 @@ export async function verifyGitWorkspaceForCreation(context: IActionContext, git
         gitWorkspaceState.repo = newRepo;
     } else if (!!gitWorkspaceState.remoteRepo && !gitWorkspaceState.hasAdminAccess) {
         context.telemetry.properties.cancelStep = 'adminAccess';
-
-        const adminAccess: string = localize('adminAccess', 'Admin access to the GitHub repository is required.  Use a repo with admin access or create a fork.');
-        await ext.ui.showWarningMessage(adminAccess, { modal: true });
-
+        const adminAccess: string = localize('adminAccess', 'Admin access to the GitHub repository "{0}" is required. Would you like to create a fork?', gitWorkspaceState.remoteRepo.name);
+        const createForkItem: MessageItem = { title: localize('createFork', 'Create Fork') };
+        await ext.ui.showWarningMessage(adminAccess, { modal: true }, createForkItem);
+
+        const repoUrl: string = (await createFork(context, gitWorkspaceState.remoteRepo)).data.html_url;
+
+        context.telemetry.properties.cancelStep = 'cloneFork';
+        const forkSuccess: string = localize('forkSuccess', 'Successfully forked "{0}". Would you like to clone your new repository?', gitWorkspaceState.remoteRepo.name);
+        const clone: MessageItem = { title: localize('clone', 'Clone Repo') };
+        ext.outputChannel.appendLog(forkSuccess);
+        void window.showInformationMessage(forkSuccess, clone).then(result => {
```

Comment:
You should be able to just use await and check the input result.  The problem with using `.then` is we'll miss the telemetry of the `afterCloneFork` because once the error is thrown, the event is sent up to telemetry.  

Normally we don't like waiting on information prompts, but due to the nature of this one (there's an action afterwards that the user has to interact with) it's okay to use.  Normally we're just like, notifying that the action has completed.